es-ds-docs/pages/molecules/dropdown.vue

```diff
@@ -0,0 +1,138 @@
+<script setup lang="ts">
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+onMounted(async () => {
+    if ($prism) {
+        const compSource = await import('@energysage/es-ds-components/components/es-dropdown.vue?raw');
+
+        const docSource = await import('./dropdown.vue?raw');
+        compCode.value = $prism.normalizeCode(compSource.default);
+        docCode.value = $prism.normalizeCode(docSource.default);
+        $prism.highlight();
+    }
+});
+
+const selectedFruit = ref<string | undefined>(undefined);
+const selectedColor = ref<string | undefined>(undefined);
+const selectedSize = ref<string | undefined>(undefined);
+
+const fruits = ['Apple', 'Banana', 'Orange', 'Grape', 'Strawberry'];
+
+const colors = ['Red', 'Orange', 'Yellow', 'Green', 'Blue', 'Indigo', 'Violet'];
+
+const sizes = ['XS', 'S', 'M', 'L', 'XL', 'XXL'];
```

Comment:
Can we have an example on this page that showcases passing in an array of objects with "option" and "value" keys, like [this example from PrimeVue](https://v3.primevue.org/dropdown/#template)? Our immediate use cases will very likely involve options having user-facing text with a different string as the value we save behind the scenes.